data/json/items/comestibles/egg.json

```diff
@@ -455,7 +455,7 @@
     "looks_like": "powder_egg",
     "quench": 2,
     "calories": 80,
-    "description": "Rehydrated eggs.  You can use it instead of an egg when cooking, provided you don't care that it has lost its complete shape.",
+    "description": "Rehydrated eggs.  You can use them instead of eggs when cooking, provided you don't care that they had lost their complete shape.",
```

Comment:
```suggestion
    "description": "Rehydrated eggs.  You can use them instead of eggs when cooking, provided you don't care that they have lost their complete shape.",
```